src/test/java/com/jcabi/beanstalk/maven/plugin/WarFileTest.java

```diff
@@ -100,4 +105,95 @@ public void checkEbextensionsValidityThrowsExceptionNoConfigFiles()
             );
         }
     }
+
+    /**
+     * Verifies that WarFile can use a .ebextensions with valid json object
+     * string. Test passes when WarFile.checkEbextensionsValidity method
+     * doesn't throw exception.
+     * @throws Exception Thrown in case of error.
+     */
+    @Test
+    public void usesEbextensionsWithValidJsonObject() throws Exception {
+        final ZipFile zip = this.zipWithEbextensionsWithText(
+            "{\"blah\":\"blah\"}"
+        );
+        final WarFile war = new WarFile(zip);
+        war.checkEbextensionsValidity();
+    }
+
+    /**
+     * Verifies that WarFile can use a .ebextensions with valid json array
+     * string. Test passes when WarFile.checkEbextensionsValidity method
+     * doesn't throw exception.
+     * @throws Exception Thrown in case of error.
+     */
+    @Test
+    public void usesEbextensionsWithValidJsonArray() throws Exception {
+        final ZipFile zip = this.zipWithEbextensionsWithText(
+            "[{\"blah\":\"blah\"},{\"blah\":\"blah\"}]"
+        );
+        final WarFile war = new WarFile(zip);
+        war.checkEbextensionsValidity();
+    }
+
+    /**
+     * Verifies that WarFile throws exception when .ebextensions
+     * contains invalid json.
+     * @todo #16 enable test when WarFile.validYaml implemented
+     * @throws Exception Thrown in case of error.
+     */
+    @Test
+    @Ignore
+    @SuppressWarnings("StringLiteralsConcatenationCheck")
+    public void throwsExceptionWhenUsesEbextensionsWithInvalidJson()
+        throws Exception {
+        final ZipFile zip = this.zipWithEbextensionsWithText(
+            "{\"blah\":\"blah\"]"
+        );
+        final WarFile war = new WarFile(zip);
+        try {
+            war.checkEbextensionsValidity();
+            Assert.fail();
+        } catch (final MojoFailureException exception) {
```

Comment:
you could just add `@Test(expected = MojoFailureException.class)` so you won't have to catch it here. The test will fail if given exception is not thrown.